spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/SparkBigQueryConfig.java

```diff
@@ -410,6 +412,8 @@ public static SparkBigQueryConfig from(
         getAnyOption(globalOptions, options, "bqPrebufferResponsesPerStream")
             .transform(Integer::parseInt)
             .or(MIN_BUFFERED_RESPONSES_PER_STREAM);
+    config.channelPoolSize =
```

Comment:
can you please add this option in `README-template.md` and in `CHANGES.md`

Reply:
Done.